portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,66 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <!-- Imported Libraries: Google's Material Design Library and Google's Fonts Library -->
+    <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
+    <title>MM Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+  <body onload="background()">
+    <div id="intro-content" style="display:block;">
+      <div id="intro-button-one" style="display:inline-block">
+        <div class="mdc-touch-target-wrapper" style="display:block;">
+          <button class="mdc-button mdc-button--outlined" onclick="intro()" style="color:white;">  
+	        <span class="mdc-button__ripple">Intro</span> 
+	      </button>
+        </div>
+        <div class="mdc-touch-target-wrapper" style="display:block;">
+	      <button class="mdc-button mdc-button--outlined" onclick="tech()" style="color:white;">  
+	        <span class="mdc-button__ripple">Tech Skills</span> 
+	      </button>
+        </div>
+        <div class="mdc-touch-target-wrapper" style="display:block;">
+	      <button class="mdc-button mdc-button--outlined" onclick="interests()" style="color:white;">  
+	        <span class="mdc-button__ripple">Interests</span>
+	      </button>
+        </div>
+      </div>
+      <div id="basic-profile" style="display:inline-block;">
+        <img src="/images/PROFILE_PIC.JPG">
+      	<h1>Michaela Murray's Portfolio</h1>
+      	<div id="intro">
+      	  <ul style="list-style-type:none;">
+            <li>Stanford Class of 2022</li> 
+            <li>Major: Computer Science</li>
+            <li>STEP Intern @ Google</li>
+	      </ul>
+        </div>
+      </div>
+      <div id="intro-button-two" style="display:inline-block;">
+	    <div class="mdc-touch-target-wrapper" style="display:block;">
+	      <button class="mdc-button mdc-button--outlined" onclick="blog()" style="color:white;">  
```

Comment:
In a similar vein to the above comment, the `blog` method doesn't exist, either. I would drop this `onclick` in this PR and then add it in a follow up.

Reply:
sounds good! Thanks!